assets/multiqc_config.yml

```diff
@@ -122,6 +123,15 @@ custom_data:
       title: "Peptide Intensity Distribution"
       description: "Comparison of log2-transformed peptide intensities across samples. Helps assess data consistency and detect potential outliers or batch effects."
       xlab: "log2(Intensity)"
+
+  epicore_length_distribution:
```

Comment:
Is it possible to add a description of the plots as well when importing htmls in multiqc?